spec/models/report_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  let(:user) do
+    User.create(
+      email: 'user@example.com',
+      name: 'テスト 太郎',
+      password: 'password',
+      password_confirmation: 'password'
+    )
+  end
+  let(:content) { 'コンテンツ' }
+  let(:report) { user.reports.create(title: 'タイトル', content:) }
+
+  describe '#editable?' do
+    context 'with 自分の日報' do
+      subject { report.editable?(user) }
+
+      it { is_expected.to eq true }
```

Comment:
```suggestion
      it { is_expected.to be_truthy }
```

って書くことが多いかなー。でもこれでもダメってことはないです

Reply:
確かにここはどちらでも良いですね。
あえて理由をつけるとしたら、 `Report#editable?(user)` は `?` で終わるメソッドなので戻り値は `true` or `false` のどちらかであることを厳格に判定した方が良いかなと思いこの書き方にしています。

一方で `be_truthy` というマッチャの存在も知っておいて欲しいので、模範解答を公開するときには別解として提示したいと思いました 👍

Reply:
> あえて理由をつけるとしたら、 Report#editable?(user) は ? で終わるメソッドなので戻り値は true or false のどちらかであることを厳格に判定した方が良いかなと思いこの書き方にしています。

参考情報ですが、true/false/nil/数値/シンボルなどは be を使うことも多いです。

https://qiita.com/jnchito/items/2e79a1abe7cd8214caa5#%E3%81%A1%E3%82%87%E3%81%A3%E3%81%A8%E3%82%84%E3%82%84%E3%81%93%E3%81%97%E3%81%84%E8%A9%B1be-%E3%81%A8-eq-%E3%81%AE%E9%81%95%E3%81%84

が、eqとbeの使い分けをFBC生に求めるのはtoo muchなので、eq trueでもいいと思いますー。